webpack.config.js

```diff
@@ -16,57 +14,21 @@ module.exports = {
 	module: {
 		rules: [{
 			test: /\.js?$/,
-			use: [
-				{
-					loader: require.resolve('babel-loader'),
-					options: {
-						compact: true,
-						highlightCode: true,
-						plugins: [
-							"@babel/plugin-transform-template-literals",
-							"@babel/plugin-transform-arrow-functions"
+			use: {
+				loader: 'babel-loader',
+				options: {
+					presets: [
+						[
+							"@babel/preset-env", {
+								"useBuiltIns": "entry",
+								"corejs": 3,
+								"debug": true,
```

Comment:
Maybe we would want "debug" turned off for distribution?  (just guessing... I'm not sure exactly what this does)

Reply:
From Joel:  debug just prints out a bunch of extra statements in the build command. It'll be nice to have that printout in the jenkins build history

I think we want to keep it then.

Reply:
Debug just prints out a bunch of extra statements in the build command. It'll be nice to have that printout in the jenkins build history